qcodes/dataset/data_set.py

```diff
@@ -1074,19 +1033,6 @@ def write_data_to_text_file(self, path: str,
                 df_to_save = pd.concat(dfs_to_save, axis=1)
                 df_to_save.to_csv(path_or_buf=dst, header=False, sep='\t')
 
-    @deprecate('This method does not accurately represent the dataset.',
-               'Use `get_parameter_data` instead.')
-    def get_values(self, param_name: str) -> List[List[Any]]:
-        """
-        Get the values (i.e. not NULLs) of the specified parameter
-        """
-        if param_name not in self.parameters:
-            raise ValueError('Unknown parameter, not in this DataSet')
-
-        values = get_values(self.conn, self.table_name, param_name)
-
-        return values
-
     def get_setpoints(self, param_name: str) -> Dict[str, List[List[Any]]]:
```

Comment:
shall we at last put a deprecation on get_setpoints, if we are not removing it now?

Reply:
Yes but let me do that in a different pr. IMHO for the future deprecting it should also involve fixing all tests that use it to either capture the warnings or not use the function so the work is likely to be a bit more involved